server/router_utils.go

```diff
@@ -0,0 +1,126 @@
+package server
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"net/http"
+	"strconv"
+
+	"github.com/RedHatInsights/insights-operator-utils/responses"
+	"github.com/RedHatInsights/insights-results-aggregator/types"
+	"github.com/google/uuid"
+	"github.com/gorilla/mux"
+)
+
+// RouterMissingParamError missing parameter in URL
+type RouterMissingParamError struct {
+	paramName string
+}
+
+func (e *RouterMissingParamError) Error() string { return fmt.Sprintf("missing param %v", e.paramName) }
```

Comment:
formatting?

Reply:
What do you mean?

Reply:
@Sergey1011010 It's rather unconventional to have an actual one-liner function in Go. I belive that's what Pavel meant here.

Reply:
@natiiix sounds reasonable, fixed